jonathaneeckhout/jdungeon
# add disclaimer panel

Adds a disclaimer panel to inform the users about the limitations of the game.

It's only activated when debug is false.

How to test:

Make sure that DEBUG is false in your .env file.
Open up the client, the panel should pop up. Press agree and the login panel should pop up.
